### DISCUSSION TITLE
COLAB %pip install lightweight_mmm error

### DISCUSSION THREAD

**Discussion by miaox018** [google/lightweight_mmm#138] [Q&A]
Did anyone have issue installing the package in colab? I previously ran successfully but now see issue below

ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
pymc 4.1.4 requires arviz>=0.12.0, but you have arviz 0.11.2 which is incompatible.
pydantic 1.10.2 requires typing-extensions>=4.1.0, but you have typing-extensions 3.10.0.2 which is incompatible.
numba 0.56.4 requires numpy<1.24,>=1.18, but you have numpy 1.24.1 which is incompatible.

And this might be related - when I run the next line `%import jax.numpy as jnp` it produces another error:
AttributeError: module 'numpy' has no attribute '_no_nep50_warning'

**Comment by michevan** [comment]
Hi @miaox018 ! 

This is probably due to the 0.1.7 version upgrade to LMMM we pushed yesterday. With the new version, the dependencies have changed, so you'll probably need to re-install LMMM in a clean environment so that it can install the correct dependencies from scratch.

**Comment by miaox018** [comment]
I created a new environment on colab but it's still not working. Could you share the code/setup that successfully run the first couple lines below? Thanks!

```
%pip install lightweight_mmm
import jax.numpy as jnp
import numpyro
numpyro.set_host_device_count(2)
```

**Reply by michevan** [reply]
are you loading your new environment within Conda? You can do this in the Colab using the %%bash magic commands, e.g if you're using conda:

%%bash
source activate myenv
